sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.FormRecognizer
+{
+    /// <summary>
+    /// </summary>
+    public class FormRecognizerClient
+    {
+        private readonly ServiceClient _serviceClient;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FormRecognizerClient"/>.
+        /// </summary>
+        public FormRecognizerClient(Uri endpoint, AzureKeyCredential credential) : this(endpoint, credential, new FormRecognizerClientOptions())
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FormRecognizerClient"/>.
+        /// </summary>
+        public FormRecognizerClient(Uri endpoint, AzureKeyCredential credential, FormRecognizerClientOptions options)
+        {
+            var diagnostics = new ClientDiagnostics(options);
+            var pipeline = HttpPipelineBuilder.Build(options, new AzureKeyCredentialPolicy(credential, Constants.AuthorizationHeader));
+            _serviceClient = new ServiceClient(diagnostics, pipeline, endpoint.ToString());
+        }
+
+        /// <summary>
+        /// </summary>
+        protected FormRecognizerClient()
+        {
+        }
+
+        #region Content
+
+        /// <summary>
+        /// Extracts layout elements from one or more passed-in forms.
```

Comment:
not necessary for this PR, but we use `extract` for documentation, but the method name is `recognize`, so it seems a little off. If it is ok, we could update the docs with `recognize`

Reply:
You have a point. I haven't updated documentation in this file yet, but it won't hurt to fix it now.

Reply:
I'll merge it now to put the new client in place, but I already have a commit ready to update that in the next PR. We'll need to revisit the whole documentation at some point, though.